src/sql/workbench/contrib/extensions/browser/scenarioRecommendations.ts

```diff
@@ -42,11 +44,15 @@ export class ScenarioRecommendations extends ExtensionRecommendations {
 		return;
 	}
 
-	// {{SQL CARBON EDIT}}
+	private ignoreRecommendations(): boolean {
+		const ignoreRecommendations = this.configurationService.getValue<boolean>('extensions.ignoreRecommendations');
```

Comment:
Not sure why a separate function is necessary for a one-line thing that's only used here

`if (this.storageService.getBoolean(storageKey, StorageScope.GLOBAL, false) || this.configurationService.getValue<boolean>('extensions.ignoreRecommendations')) {`

It's not really that much longer just to put it in there